lib/simp/cli/commands/bootstrap.rb

```diff
@@ -1,66 +1,335 @@
+              system(%{cp #{f} #{f}.BAK})
+              say "> Sucessfully backed up #{f} to #{f}.BAK".green
+            end
+          end
+        else
+          fail( "Could not find directory #{puppetserver_dir}" )
+        end
+
+        # Run in a temporary cache space.
+        server_conf_tmp = "#{::Utils.puppet_info[:config]['vardir']}/pserver_tmp"
+        FileUtils.mkdir_p(server_conf_tmp)
+        FileUtils.chown('puppet','puppet',server_conf_tmp)
+        system(%{puppet resource simp_file_line puppetserver path='/etc/sysconfig/puppetserver' match='^JAVA_ARGS' line='JAVA_ARGS="-Xms2g -Xmx2g -XX:MaxPermSize=256m -Djava.io.tmpdir=#{server_conf_tmp}"' 2>&1 > /dev/null})
+        say "> Sucessfully wrote java tmpdir to /etc/sysconfig/puppetserver".green
+
+        # Slap minimalistic conf files in place to get puppetserver off of the ground.
+        system(%{cat > #{puppetserver_dir}/webserver.conf <<-EOM
+webserver: {
+    access-log-config: /etc/puppetlabs/puppetserver/request-logging.xml
+    client-auth: want
+    ssl-host = 0.0.0.0
+    ssl-port = 8150
+}
+EOM
+})
+        say "> Sucessfully wrote webserver.conf to #{puppetserver_dir}/webserver.conf".green
+        system(%{cat > #{puppetserver_dir}/web-routes.conf <<-EOM
+web-router-service: {
+    "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
+    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service": ""
+    "puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service": "/puppet-admin-api"
+    "puppetlabs.trapperkeeper.services.status.status-service/status-service": "/status"
+}
+EOM
+})
+        say "> Sucessfully wrote web-routes.conf to #{puppetserver_dir}/web-routes.conf".green
+      rescue => error
+        fail( "Failed to configure the puppetserver, with error #{error.message}" )
+      end
+
+      # Firstrun is tagged and run against the bootstrap puppetserver port, 8150.
+      # This run will configure puppetserver and puppetdb; all subsequent runs
+      # will run against the conifgured masterport.
+      say "> Running puppet agent, with --tags pupmod,simp".cyan
+      pupcmd = 'puppet agent --onetime --no-daemonize --no-show_diff --verbose --no-splay --masterport=8150 --ca_port=8150'
+      # Firstrun is tagged and run against the bootstrap puppetserver port, 8150.
+      linecounts << track_output("#{pupcmd} --tags pupmod,simp 2> /dev/null", '8150')
+
+      # If selinux is enabled, relabel the filesystem.
+      FileUtils.touch('/.autorelabel')
+      if Facter.value(:selinux) && !Facter.value(:selinux_current_mode).nil? && (Facter.value(:selinux_current_mode) != "disabled")
+        say "> Relabeling filesystem for selinux".cyan
+        @logfile.puts("Relabeling filesystem for selinux.\n")
+        system("fixfiles -f relabel 2>&1 | tee -a #{@logfile.path}")
```

Comment:
Why are we showing the stdout/stderr output of fixfiles?  I would only  expect to see stderr on the console and would expect stdout and stderr to both go to the log.